Chapter1/Zad4_.py

```diff
@@ -0,0 +1,23 @@
+'''
+Created on 28 sty 2018
+
+@author: agata.wiewiora
+
+This program that reads an integer number and prints its 
+previous and next numbers.
+
+'''
+
+
+# Read an integer:
+integer_ = input("Please, provide sample integer: ")
+if '.' in str(integer_):
```

Comment:
sprytnie, ale jeżeli chcesz sprawdzić czy zmienna jest jakiegoś typu to lepszym pomysłem jest użycie funkcji isinstance lub type niż castowanie i szukanie kropek:)